docs/Monitored_Only_ZFS.md

```diff
@@ -0,0 +1,208 @@
+
+The lustre filesystem will be created from the command line on zpools and IML GUI will be used to scan for the filesytem.
+Note that VM Disks (ata-VBOX_HARDDISK...) will be mapped as /dev/sd devices. Use "ls -l /dev/disk/by-id/" to list the VM Disks available. 
+
+- Management Target:
+```
+    vagrant ssh mds1
+    sudo -s
+    #note use /dev/sdb (512M) for the mgt
+    zpool create mgs -o cachefile=none /dev/disk/by-id/ata-VBOX_HARDDISK_VBe85051d4-e6ae953d -f
+    mkfs.lustre --reformat --failover 10.73.20.12@tcp --mgs --backfstype=zfs mgs/mgt
+    mkdir -p /lustre/mgs
+    mount -t lustre mgs/mgt /lustre/mgs
+```
+
+- Metadata Target:
+```
+    vagrant ssh mds2
+    sudo -s
+    zpool create mds -o cachefile=none /dev/disk/by-id/ata-VBOX_HARDDISK_VB0729fac1-5420a643 -f
+    mkfs.lustre --reformat --failover 10.73.20.11@tcp  --mdt --backfstype=zfs --fsname=zfsmo --index=0 --mgsnode=10.73.20.11@tcp mds/mdt0
+    mkdir -p /lustre/zfsmo/mdt0
+    mount -t lustre mds/mdt0 /lustre/zfsmo/mdt0
+```
+- Object Storage Targets:
+```
+    vagrant ssh oss1
+    sudo -s
+    zpool create oss1 -o cachefile=none raidz2 /dev/disk/by-id/ata-VBOX_HARDDISK_VB6f41df02-0d5d2a15 /dev/disk/by-id/ata-VBOX_HARDDISK_VB06b563a9-2539af7b  /dev/disk/by-id/ata-VBOX_HARDDISK_VBb2ec79fb-b900b724 /dev/disk/by-id/ata-VBOX_HARDDISK_VBe2585d74-5267121b -f
+    mkfs.lustre --reformat --failover 10.73.20.22@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=0 --mgsnode=10.73.20.11@tcp oss1/ost00
+    zfs compression=on oss1
+    mkdir -p /lustre/zfsmo/ost00
+    mount -t lustre oss1/ost00 /lustre/zfsmo/ost00
+
+    vagrant ssh oss2
+    sudo -s
+    zpool create oss2 -o cachefile=none raidz2 /dev/disk/by-id/ata-VBOX_HARDDISK_VBb7776d10-aba16176 /dev/disk/by-id/ata-VBOX_HARDDISK_VB6657241f-bab2ffed  /dev/disk/by-id/ata-VBOX_HARDDISK_VB0ebfc209-9bbf80c4 /dev/disk/by-id/ata-VBOX_HARDDISK_VB51449aa2-95df9cdc -f
+    mkfs.lustre --reformat --failover 10.73.20.21@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=1 --mgsnode=10.73.20.11 oss2/ost01
+    zfs compression=on oss2
+    mkdir -p /lustre/zfsmo/ost01
+    mount -t lustre oss2/ost01 /lustre/zfsmo/ost01
+```
+
+After all the commands for each had run successfully, use the IML GUI to scan for the filesystem:
+    Configuration -> Servers -> Scan for Filesystem.  Use all servers
+
+If Succussfull, in the IML GUI, the filesystem will be available.
+
+## Setting up Clients
+In your vagrant folder, run the following script to prepare both client c1 and c2:
+```
+vagrant sh -c '\
+sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/lustre-client/repo/epel-7/managerforlustre-lustre-client-epel-7.repo && \
+sudo yum-config-manager --add-repo https://downloads.hpdd.intel.com/public/e2fsprogs/latest/el7/ && \
+sudo sed -i -e "1d" -e "2s/^.*$/[e2fsprogs]/" -e "/baseurl/s/,/%2C/g" -e "/enabled/a gpgcheck=0" /etc/yum.repos.d/downloads.hpdd.intel.com_public_e2fsprogs_latest_el7_.repo && \
```

Comment:
None of these repos are needed for clients.

Reply:
Fixed